setup.py

```diff
@@ -597,10 +618,21 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
     'virtualenv': virtualenv,
 }
 
-# Add extras for all providers. For all providers the extras name = providers name
-for provider_name, provider_requirement in PROVIDERS_REQUIREMENTS.items():
-    EXTRAS_REQUIREMENTS[provider_name] = provider_requirement
+
+def add_extras_for_all_providers() -> None:
+    """
+    Adds extras for all providers.
+    By default all providers have the same extra name as provider id, for example
+    'apache.hive' extra has 'apache.hive' provider requirement.
+    :return:
```

Comment:
```suggestion
```

Since it is empty it is better to remove it.

Reply:
Yep